README.adoc

```diff
@@ -172,10 +173,17 @@ for SSL certificate validation
 Supported schemes are `http` and `https`.
 * `connection-timeout` sets the connection timeout in milliseconds
 * `read-timeout` sets the read timeout in milliseconds
+* `token` sets the main token to authenticate with vault.  This can also be set with the environment variable SPRING_CLOUD_VAULT_TOKEN
```

Comment:
I missed one point, sorry for that: `README.md` is generated from `docs/src/main/asciidoc/README.adoc` using `docs/src/main/ruby/generate_readme.sh`. If you want to contribute documentation changes, please do these in the documentation sources in `docs/src/main/asciidoc/`. I added a reference to the reference documentation in the meantime.